Chinese/Demo_htx/demo.md

```diff
@@ -0,0 +1,2 @@
+click [here](http://www.baidu.com) to visit baidu
+click [here](https://github.com/udacity/create-your-own-adventure) to visit raw url
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.